specification/cognitiveservices/Language.Conversations/tsp-output/@azure-tools/typespec-autorest/azure.ai.language.conversations.json

```diff
@@ -34,24 +34,22 @@
   ],
   "security": [
     {
-      "apim_key": []
```

Comment:
Since the alias doesn't work, just keep passing this as you were and I'll figure out if there's a way - or even reason to - rename the "OAuth2Auth" def.